Azure.Data.Wrappers/StorageQueue.cs

```diff
@@ -177,19 +195,30 @@ public virtual CloudQueue Reference
         /// <returns>Task</returns>
         public virtual async Task Send(object obj)
         {
-            if (null == obj)
+            await this.SendAsync(obj);
+        }
+
+        /// <summary>
+        /// Save Model to queue, as json
+        /// </summary>
+        /// <param name="message">object</param>
+        /// <returns>Task</returns>
+        public virtual async Task SendAsync<T>(T message)
+        {
+            if (null == message)
```

Comment:
I realize this was already there, but this seems to be doing more than wrapping.  Is that the expected behavior?

Reply:
it's doing the right thing is the message is just a CloudQueueMessage and serializing it if not.